src/components/Element.js

```diff
@@ -28,8 +28,8 @@ const _extractOptions = (props: Props): Object => {
   return options;
 };
 
-const Element = (type: string, hocOptions: {sourceType?: string} = {}) =>
-  class extends React.Component {
+const Element = (type: string, hocOptions: {sourceType?: string} = {}): * =>
```

Comment:
I think it's best to avoid using `*` operator whenever possible. It tends to either do the same thing as not having it or something unintuitive.

In this example, it's unnecessary because inference tells Flow the return type. If you want to be explicit, you can try
```
: React.ComponentType<Props>
```

Reply:
The problem in this case is, that adding the explicit return type does not work:
https://flow.org/try/#0JYWwDg9gTgLgBAKjgQwM5wEoFNkGN4BmUEIcA5FDvmQNwBQdMAnmFnAArFjoC8cA3nThwCECAC44AOwCuIAEZYoAGiFx5yKJNkKlq4RoBe2uYpV0AvvTq4IU1PACiAGywgsU+HwAUdAJDMrJIOUMBSAOZ0AJSS2HgwAHQAwiSQUh4wADycENwAfHA8BYLClDAyUFJwuM5o6FgAHjAeACbocfjJqXYZ2VyoxWrCDsgwwLhwLVgEyDLOMDnchQJDwiJikgCM+msGmpIATDu7RpIAzMdwVquUUlNQ3lEru6VY5ZVwmS3AAG5wAPR5ei7Cxqa6gmx2BxwACyTBcbgyywR7k83jIIDCWDIUWs-3+cAAygALCBzFrSCDwJTEKCTGRsGAQSbTWbzOBgfoJOiZOEopGAvEEklk5wUmnQemM5kAd2IETggSwPL5rlRhDEPDIzQcZABQLoQA

And adding the `*` seemed to look more intentional than not having a return type. I am fine with removing it, though!